### PR TITLE
Features/refactor

### PR DESCRIPTION
**Pull request by mateuszorman**

### CODE REVIEW

**Comment by mateuszorman** [comment]
Please take a look at my comments. Thanks!

File: szkolenie/szkolenie/Lesson1.cs [hunk 1]
```diff
@@ -9,28 +9,35 @@
...
 using OpenQA.Selenium.Support.UI;
 
-namespace Szkolenie
+namespace szkolenie
 {
-    public class Lesson1 : szkolenie.Fixtures
+    public class Lesson1 : Fixtures
     {
 
         [Test]
```
Comment (brozmus) [inline szkolenie/szkolenie/Lesson1.cs]: Having [TestCase] you don't need [Test]. It's redundant now.

File: szkolenie/szkolenie/Lesson1.cs [hunk 2]
```diff
@@ -9,28 +9,35 @@
...
+            // act
             eneterMessage.SendKeys(typed_text);
             buttons[0].Click();
             string read_text = textElem.Text;
+
+            //assert
             Assert.AreEqual(read_text, typed_text);
         }
 
         [Test]
```
Comment (brozmus) [inline szkolenie/szkolenie/Lesson1.cs]: Having [TestCase] you don't need [Test]. It's redundant now.

File: szkolenie/szkolenie/Lesson2.cs [hunk 3]
```diff
@@ -9,70 +9,59 @@
...
 using OpenQA.Selenium.Support.UI;
 
-namespace Szkolenie
+namespace szkolenie
 {
-    public class Lesson2
+    public class Lesson2 : Fixtures
     {
-        ChromeDriver driver;
         [Test]
```
Comment (brozmus) [inline szkolenie/szkolenie/Lesson2.cs]: Having [TestCase] you don't need [Test]. It's redundant now.

File: szkolenie/szkolenie/Lesson2.cs [hunk 4]
```diff
@@ -9,70 +9,59 @@
...
             var option = driver.FindElement(By.CssSelector("ul+ .checkbox label"));
             var singleCheckBoxBTN = driver.FindElement(By.XPath("/html/body/div[2]/div/div[2]/div[1]/div[2]/div[1]/label"));
 
+            // act
             singleCheckBoxBTN.Click();
             option.Click();
             logoText.Click();
 
-            driver.Quit();
+            // assert
```
Comment (brozmus) [inline szkolenie/szkolenie/Lesson2.cs:33]: emtpy section?

File: szkolenie/szkolenie/Lesson3.cs [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+            Assert.AreEqual(newZealandString, selectCountryDD.Text);
+            Assert.IsTrue(selectCountryDD.Text == newZealandString);
+            Assert.That(selectCountryDD.Text == newZealandString);
+        }
+
+        [Test]
+        [TestCase("https://www.seleniumeasy.com/test/basic-first-form-demo.html", "false")]
+        public void ExplicitWait(string URL, string startup_popup)
+        {
+            // arrange
```
Comment (brozmus) [inline szkolenie/szkolenie/Lesson3.cs:39]: empty section?
You can put here assignment of popup.

File: szkolenie/szkolenie/Lesson3.cs [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+            // act
+            var wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+            wait.Until(d => d.FindElement(By.Id("at-cv-lightbox-close")));
+            var Popup = driver.FindElement(By.Id("at-cv-lightbox-close"));
```
Comment (brozmus) [inline szkolenie/szkolenie/Lesson3.cs]: local variables in camel case

File: szkolenie/szkolenie/Lesson3.cs [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+            Popup.Click();
+
+            // assert
+            try
+            {
+                Assert.IsFalse(Popup.Displayed);
+            }
+            catch (Exception e)
+            {
+                // How to log.info?
```
Comment (brozmus) [inline szkolenie/szkolenie/Lesson3.cs:55]: Maybe Assert.Fail(message)?

File: szkolenie/szkolenie/Lesson3.cs [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+            }
+        }
+
+        [Test]
+        [TestCase("https://www.globalsqa.com/demo-site/draganddrop/", "false")]
+        public void DragAndDrop(string URL, string startup_popup)
+        {
+            // arrange
+            var frame = driver.FindElement(By.ClassName("demo-frame"));
+            // act
```
Comment (brozmus) [inline szkolenie/szkolenie/Lesson3.cs:65]: Add new line before this comment.

**Review by brozmus — CHANGES_REQUESTED** [review]

File: szkolenie/szkolenie/Lesson3.cs (hunk 7 above)
Comment (mateuszorman) [reply szkolenie/szkolenie/Lesson3.cs:55]: I want to put log just for information that popup is not visible anymore.

**Review by brozmus — APPROVED** [review]

**Pull request merged**